Src/library/memory_analyzer/memory_analyzer.ecf

```diff
@@ -24,4 +21,9 @@
 		<cluster name="stone" location="stone\" recursive="true"/>
 		<cluster name="widgets" location="widgets\" recursive="true"/>
 	</target>
+	<target name="mem" extends="memory_analyzer">
```

Comment:
no I removed it, this was to compile in non void-safe mode while converting.
This won't be committed.